src/compute/src/render/reduce.rs

```diff
@@ -93,8 +91,12 @@ where
                     val_plan.permute(demand_map, demand_map_len);
                     let skips = mz_compute_types::plan::reduce::convert_indexes_to_skips(demand);
                     move |row_datums, time, diff| {
+                        let binding = SharedRow::get();
+                        let mut row_builder = binding.borrow_mut();
                         let temp_storage = RowArena::new();
 
+                        println!("{:?}", row_datums);
```

Comment:
Stray?

Reply:
Yes! That's a leftover from some testing :)